packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -1,45 +1,374 @@
 <template>
-    <component
-        :is="componentVersion"
-        v-bind="data"
-        :flags="flags"
-        :error-boundary="errorBoundary">
-        <template
-            v-for="(_, slotName) in $slots"
-            v-slot:[slotName]>
-            <slot :name="slotName" />
-        </template>
-    </component>
+    <a
+        :href="url"
+        :class="[
+            $style['c-restaurantCard'], {
+                [$style['c-restaurantCard--listItem']]: isListItem,
```

Comment:
I wonder if maybe this class should be `c-restaurantCard--item` or another name? For me there's a bit of a dissonance in using list item as a name when the element is `<a>` rather than a `<li>`

Reply:
yeah it's a tricky one - this is for when they are on desktop, but I'd like to avoid using the name 'desktop'. list item is certainly confusing, and I think 'item' might be equally ambiguous. I'll have a think on a better name for this and update

Reply:
<img width="743" alt="Screenshot 2022-02-15 at 11 57 12" src="https://user-images.githubusercontent.com/16766185/154057686-5c7bf292-8569-44b8-ab1b-be73cdb327c7.png">


@ashleynolan the `listItem` modifier is the term we dubbed for the desktop style listing you can see in the image attached

Reply:
(the one on the right)

Reply:
Hmm, naming is hard 😆 

I think as long as it makes sense to you all, that's the main thing tbh. Could use something like `c-restaurantCard--stretchPanel` or something equally vague to describe it 🤷

Reply:
Indeed it is 😆 ok cool - as @AlixRichards rightly pointed out, listItem is not the most understandable term. I'll see what she thinks of stretchPanel and will happily change if so!

Reply:
`stretchPanel` is potentially cool, I'm still unclear what the modifier does even seeing the image 🤔 
Can you remind me what this looks like on desktop without the modifier 🤞

Reply:
@AlixRichards the modifier makes it look like the right hand image above. Otherwise it's just the left hand image but full width :)